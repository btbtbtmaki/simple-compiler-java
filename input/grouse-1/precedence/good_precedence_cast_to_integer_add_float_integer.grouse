main {

  print 7.3 :int + 99 nl;
  print 1.3 :int + 26 nl;
  print 4.6 :int + 81 nl;
  print 7.4 :int + 37 nl;
  print 4.9 :int + 80 nl;
  print 1.6 :int + 50 nl;
  print 9.0 :int + 23 nl;
  print 3.9 :int + 64 nl;
  print 6.2 :int + 72 nl;
  print 6.2 :int + 34 nl;
  print 6.7 :int + 60 nl;
  print 9.1 :int + 26 nl;

  print 47 + 8.5 :int nl;
  print 95 + 1.2 :int nl;
  print 54 + 8.5 :int nl;
  print 92 + 5.0 :int nl;
  print 81 + 8.7 :int nl;
  print 45 + 9.6 :int nl;
  print 75 + 7.1 :int nl;
  print 14 + 2.6 :int nl;
  print 27 + 4.6 :int nl;
  print 26 + 8.0 :int nl;
  print 75 + 8.3 :int nl;
  print 54 + 8.6 :int nl;

  print 9.4 :int + 6.4 :int nl;
  print 9.4 :int + 7.5 :int nl;
  print 4.0 :int + 9.7 :int nl;
  print 9.6 :int + 7.4 :int nl;
  print 4.0 :int + 3.5 :int nl;
  print 7.3 :int + 4.3 :int nl;
  print 2.7 :int + 5.9 :int nl;
  print 1.7 :int + 8.6 :int nl;
  print 9.8 :int + 1.1 :int nl;
  print 3.1 :int + 5.3 :int nl;
  print 7.3 :int + 2.1 :int nl;
  print 7.9 :int + 7.0 :int nl;

}

