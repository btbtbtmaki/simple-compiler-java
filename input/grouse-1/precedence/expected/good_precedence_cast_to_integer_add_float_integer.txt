106
27
85
44
84
51
32
67
78
40
66
35
55
96
62
97
89
54
82
16
31
34
83
62
15
16
13
16
7
11
7
9
10
8
9
14
